charts/velero/templates/deployment.yaml

```diff
@@ -122,6 +122,8 @@ spec:
       initContainers:
         {{- toYaml .Values.initContainers | nindent 8 }}
 {{- end }}
+      securityContext:
+        {{- toYaml .Values.securityContext | nindent 8 }}
```

Comment:
Thanks for adding this.  It would also be nice to add this to the restic DaemonSet as well, for parity.

This and a rebase and it should be good to go!